python/az/aro/azext_aro/_params.py

```diff
@@ -160,6 +161,9 @@ def load_arguments(self, _):
                    options_list=['--assign-platform-workload-identity', '--assign-platform-wi'],
                    validator=validate_platform_workload_identities(isCreate=False),
                    action=AROPlatformWorkloadIdentityAddAction, nargs='+')
+        c.argument('upgradeable_to', arg_group='Identity', options_list=['--upgradeable-to'],
+        help='OpenShift version to upgrade/update to.', is_preview=True,
+        validator=validate_upgradeable_to_format)
```

Comment:
I believe the indentation is incorrect here.

Reply:
this doesn't appear to break functionality at all, but I agree with fixing this before merge.